components/sendgrid/actions/list-global-supressions/list-global-supressions.js

```diff
@@ -0,0 +1,75 @@
+const validate = require("validate.js");
+const common = require("../common");
+
+module.exports = {
+  ...common,
+  key: "sendgrid-list-global-suppressions",
+  name: "List Global Suppressions",
+  description:
+    "Allows you to get a list of all email address that are globally suppressed.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    startTime: {
+      type: "integer",
+      label: "Start Time",
+      description:
+        "Refers start of the time range in unix timestamp when an unsubscribe email was created (inclusive).",
+      optional: true,
+    },
+    endTime: {
+      type: "integer",
+      label: "End Time",
+      description:
+        "Refers end of the time range in unix timestamp when an unsubscribe email was created (inclusive).",
+      optional: true,
+    },
+    numberOfSupressions: {
+      type: "integer",
+      label: "Max # of Global Suppressions to Return",
+      description: "Indicates the max number of global suppressions to return.",
+    },
+  },
+  methods: {
+    ...common.methods,
+  },
+  async run() {
```

Comment:
Include the `$` variable.
```suggestion
  async run({ $ }) {
```